tests/integration/support/client_wrapper.go

```diff
@@ -75,9 +77,20 @@ func GetInstance(datafileName string) *ClientWrapper {
 		Datafile: datafile,
 	}
 
-	decisionService := &optlyplugins.TestCompositeService{CompositeService: *decision.NewCompositeService("")}
 	eventProcessor.EventDispatcher = &optlyplugins.ProxyEventDispatcher{}
 
+	config, err := configManager.GetConfig()
+	if err != nil {
+		log.Fatal(err)
+	}
+	userProfileService := createUserProfileService(config, apiOptions)
+
+	experimentServiceOptions := []decision.CESOptionFunc{}
+	experimentServiceOptions = append(experimentServiceOptions, decision.WithUserProfileService(userProfileService))
+	compositeExperimentService := decision.NewCompositeExperimentService(experimentServiceOptions...)
+	compositeService := *decision.NewCompositeService("", decision.WithCompositeExperimentService(compositeExperimentService))
```

Comment:
If there is an SDK key provided (not empty string), then we should use it, otherwise we won't be able to grab the notification center properly

Reply:
We haven't yet implemented support for DFM tests, once we do, this will be updated with the provided `sdkKey`.

Reply:
ok can you just add a TODO here about it